### ISSUE TITLE
Fails to identify devicePixelRatio and make a screenshot on IE

### ISSUE THREAD

**Issue by ishubin** [galenframework/galen#282]
```
========================================
Test: MyProject on desktop device in ie browser
========================================
Started InternetExplorerDriver server (64-bit)
2.39.0.0
Listening on port 34678
    check specs/myProject.spec --include "desktop"
[pool-1-thread-1] ERROR net.mindengine.galen.api.Galen - Error during setting sc
reenshot.

java.lang.RuntimeException: Error making screenshot
        at net.mindengine.galen.browser.SeleniumBrowser.createScreenshot(Seleniu
mBrowser.java:87)
        at net.mindengine.galen.page.selenium.SeleniumPage.createScreenshot(Sele
niumPage.java:231)
        at net.mindengine.galen.api.Galen.checkLayoutForPage(Galen.java:95)
        at net.mindengine.galen.api.Galen.checkLayout(Galen.java:81)
        at net.mindengine.galen.api.Galen.checkLayout(Galen.java:65)
        at net.mindengine.galen.api.Galen.checkLayout(Galen.java:51)
        at net.mindengine.galen.suite.actions.GalenPageActionCheck.execute(Galen
PageActionCheck.java:46)
        at net.mindengine.galen.runner.GalenPageRunner.executeAction(GalenPageRu
nner.java:77)
        at net.mindengine.galen.runner.GalenPageRunner.run(GalenPageRunner.java:
69)
        at net.mindengine.galen.runner.GalenBasicTestRunner.runTest(GalenBasicTe
stRunner.java:71)
        at net.mindengine.galen.tests.GalenBasicTest.execute(GalenBasicTest.java
:53)
        at net.mindengine.galen.TestRunnable.runTest(TestRunnable.java:68)
        at net.mindengine.galen.TestRunnable.run(TestRunnable.java:94)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: org.openqa.selenium.WebDriverException: JavaScript error (WARNING: Th
e server did not provide any stacktrace information)
Command duration or timeout: 23 milliseconds
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreat
eProcessApi=false, ie.usePerProcessProxy=false, ignoreZoomSetting=false, handles
Alerts=true, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSess
ion=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindo
wFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:
34678/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSetting
s=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true, unexpectedAle
rtBehaviour=dismiss}]
Session ID: 547d4e90-8482-4808-8f5b-3503e6613f55
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)

        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)

        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Sou
rce)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.
java:204)
        at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHa
ndler.java:156)
        at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.ja
va:599)
        at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDri
ver.java:508)
        at net.mindengine.galen.utils.GalenUtils.resizeScreenshotIfNeeded(GalenU
tils.java:173)
        at net.mindengine.galen.utils.GalenUtils.takeScreenshot(GalenUtils.java:
292)
        at net.mindengine.galen.browser.SeleniumBrowser.makeSimpleScreenshot(Sel
eniumBrowser.java:92)
        at net.mindengine.galen.browser.SeleniumBrowser.createScreenshot(Seleniu
mBrowser.java:85)
        ... 15 more
```

**Comment by sachinkodagali** [comment]
On which version of galen was this fixed .. Currently we are able to repro this in 1.6.4 with IE11

**Comment by sachinkodagali** [comment]
We also installed galen 2.1.2. We ran a sample spec file (that @ishubin had given as a fix for us last time and had worked on FF, 2.1.0 galen), first time it worked and didn't give the above error. However subsequent runs on IE11 kept giving the same issue. Error details below 

org.openqa.selenium.NoSuchWindowException: Unable to get browser (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'kodagali-PC', ip: '10.0.2.15', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_45'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=false, handlesAlerts=true, version=11, plat
ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:45897/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=true, en
ss}]
Session ID: deba27cf-9dc3-4780-91e4-9a031d96ad4d
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
        at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
        at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:595)
        at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:504)
        at com.galenframework.utils.GalenUtils.resizeScreenshotIfNeeded(GalenUtils.java:184)
        at com.galenframework.utils.GalenUtils.takeScreenshot(GalenUtils.java:306)
        at com.galenframework.browser.SeleniumBrowser.makeSimpleScreenshot(SeleniumBrowser.java:101)
        at com.galenframework.browser.SeleniumBrowser.createScreenshot(SeleniumBrowser.java:94)
        at com.galenframework.page.selenium.SeleniumPage.createScreenshot(SeleniumPage.java:171)
        at com.galenframework.api.Galen.checkLayoutForPage(Galen.java:104)
        at com.galenframework.api.Galen.checkLayout(Galen.java:90)
        at com.galenframework.api.Galen.checkLayout(Galen.java:73)
        at com.galenframework.javascript.GalenJsApi.checkLayout(GalenJsApi.java:100)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:126)
        at org.mozilla.javascript.NativeJavaMethod.call(NativeJavaMethod.java:225)
        at org.mozilla.javascript.optimizer.OptRuntime.callN(OptRuntime.java:52)
        at org.mozilla.javascript.gen._cmd__18._c_checkLayout_7(<cmd>:136)
        at org.mozilla.javascript.gen._cmd__18.call(<cmd>)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:63)
        at org.mozilla.javascript.gen.C__Users_kodagali_ASCORP_Downloads_galen_bin_2_1_2_galen_bin_2_1_2_chimeratests_ie_js_20._c_anonymous_1(C:\Users\kodagali.ASCORP\Downloads\galen-bin-2.1.2\galen-bin-2.1.2\chime
        at org.mozilla.javascript.gen.C__Users_kodagali_ASCORP_Downloads_galen_bin_2_1_2_galen_bin_2_1_2_chimeratests_ie_js_20.call(C:\Users\kodagali.ASCORP\Downloads\galen-bin-2.1.2\galen-bin-2.1.2\chimeratests_ie
        at org.mozilla.javascript.ScriptRuntime.applyOrCall(ScriptRuntime.java:2429)
        at org.mozilla.javascript.BaseFunction.execIdCall(BaseFunction.java:269)
        at org.mozilla.javascript.IdFunctionObject.call(IdFunctionObject.java:97)
        at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:32)
        at org.mozilla.javascript.gen._cmd__17._c_invokeFunc_2(<cmd>:31)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:63)
        at org.mozilla.javascript.gen._cmd__17._c_anonymous_22(<cmd>:280)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:394)
        at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3090)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.JavaAdapter.doCall(JavaAdapter.java:606)
        at org.mozilla.javascript.JavaAdapter.access$000(JavaAdapter.java:15)
        at org.mozilla.javascript.JavaAdapter$1.run(JavaAdapter.java:586)
        at org.mozilla.javascript.Context.call(Context.java:489)
        at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:504)
        at org.mozilla.javascript.JavaAdapter.callMethod(JavaAdapter.java:583)
        at adapter1.execute(<adapter>)
        at com.galenframework.TestRunnable.runTest(TestRunnable.java:75)
        at com.galenframework.TestRunnable.run(TestRunnable.java:101)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)

[pool-1-thread-1] ERROR com.galenframework.api.Galen - Error during setting screenshot.
java.lang.RuntimeException: Error making screenshot
        at com.galenframework.browser.SeleniumBrowser.createScreenshot(SeleniumBrowser.java:96)
        at com.galenframework.page.selenium.SeleniumPage.createScreenshot(SeleniumPage.java:171)
        at com.galenframework.api.Galen.checkLayoutForPage(Galen.java:104)
        at com.galenframework.api.Galen.checkLayout(Galen.java:90)
        at com.galenframework.api.Galen.checkLayout(Galen.java:73)
        at com.galenframework.javascript.GalenJsApi.checkLayout(GalenJsApi.java:100)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:126)
        at org.mozilla.javascript.NativeJavaMethod.call(NativeJavaMethod.java:225)
        at org.mozilla.javascript.optimizer.OptRuntime.callN(OptRuntime.java:52)
        at org.mozilla.javascript.gen._cmd__18._c_checkLayout_7(<cmd>:136)
        at org.mozilla.javascript.gen._cmd__18.call(<cmd>)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:63)
        at org.mozilla.javascript.gen.C__Users_kodagali_ASCORP_Downloads_galen_bin_2_1_2_galen_bin_2_1_2_chimeratests_ie_js_20._c_anonymous_1(C:\Users\kodagali.ASCORP\Downloads\galen-bin-2.1.2\galen-bin-2.1.2\chime
        at org.mozilla.javascript.gen.C__Users_kodagali_ASCORP_Downloads_galen_bin_2_1_2_galen_bin_2_1_2_chimeratests_ie_js_20.call(C:\Users\kodagali.ASCORP\Downloads\galen-bin-2.1.2\galen-bin-2.1.2\chimeratests_ie
        at org.mozilla.javascript.ScriptRuntime.applyOrCall(ScriptRuntime.java:2429)
        at org.mozilla.javascript.BaseFunction.execIdCall(BaseFunction.java:269)
        at org.mozilla.javascript.IdFunctionObject.call(IdFunctionObject.java:97)
        at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:32)
        at org.mozilla.javascript.gen._cmd__17._c_invokeFunc_2(<cmd>:31)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:63)
        at org.mozilla.javascript.gen._cmd__17._c_anonymous_22(<cmd>:280)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:394)
        at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3090)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.JavaAdapter.doCall(JavaAdapter.java:606)
        at org.mozilla.javascript.JavaAdapter.access$000(JavaAdapter.java:15)
        at org.mozilla.javascript.JavaAdapter$1.run(JavaAdapter.java:586)
        at org.mozilla.javascript.Context.call(Context.java:489)
        at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:504)
        at org.mozilla.javascript.JavaAdapter.callMethod(JavaAdapter.java:583)
        at adapter1.execute(<adapter>)
        at com.galenframework.TestRunnable.runTest(TestRunnable.java:75)
        at com.galenframework.TestRunnable.run(TestRunnable.java:101)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: image == null!
        at javax.imageio.ImageTypeSpecifier.createFromRenderedImage(Unknown Source)
        at javax.imageio.ImageIO.getWriter(Unknown Source)
        at javax.imageio.ImageIO.write(Unknown Source)
        at com.galenframework.rainbow4j.Rainbow4J.saveImage(Rainbow4J.java:294)
        at com.galenframework.utils.GalenUtils.takeScreenshot(GalenUtils.java:308)
        at com.galenframework.browser.SeleniumBrowser.makeSimpleScreenshot(SeleniumBrowser.java:101)
        at com.galenframework.browser.SeleniumBrowser.createScreenshot(SeleniumBrowser.java:94)
        ... 41 more
= Main Section =
    buyingInfoTab:
org.mozilla.javascript.WrappedException: Wrapped org.openqa.selenium.NoSuchWindowException: Unable to find elements on closed window (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'kodagali-PC', ip: '10.0.2.15', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_45'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=false, handlesAlerts=true, version=11, plat
ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:45897/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=true, en
ss}]
Session ID: deba27cf-9dc3-4780-91e4-9a031d96ad4d
*** Element info: {Using=css selector, value=div[data-tab="buy"]} (<cmd>#136)
        at org.mozilla.javascript.Context.throwAsScriptRuntimeEx(Context.java:1754)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:148)
        at org.mozilla.javascript.NativeJavaMethod.call(NativeJavaMethod.java:225)
        at org.mozilla.javascript.optimizer.OptRuntime.callN(OptRuntime.java:52)
        at org.mozilla.javascript.gen._cmd__18._c_checkLayout_7(<cmd>:136)
        at org.mozilla.javascript.gen._cmd__18.call(<cmd>)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:63)
        at org.mozilla.javascript.gen.C__Users_kodagali_ASCORP_Downloads_galen_bin_2_1_2_galen_bin_2_1_2_chimeratests_ie_js_20._c_anonymous_1(C:\Users\kodagali.ASCORP\Downloads\galen-bin-2.1.2\galen-bin-2.1.2\chime
        at org.mozilla.javascript.gen.C__Users_kodagali_ASCORP_Downloads_galen_bin_2_1_2_galen_bin_2_1_2_chimeratests_ie_js_20.call(C:\Users\kodagali.ASCORP\Downloads\galen-bin-2.1.2\galen-bin-2.1.2\chimeratests_ie
        at org.mozilla.javascript.ScriptRuntime.applyOrCall(ScriptRuntime.java:2429)
        at org.mozilla.javascript.BaseFunction.execIdCall(BaseFunction.java:269)
        at org.mozilla.javascript.IdFunctionObject.call(IdFunctionObject.java:97)
        at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:32)
        at org.mozilla.javascript.gen._cmd__17._c_invokeFunc_2(<cmd>:31)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:63)
        at org.mozilla.javascript.gen._cmd__17._c_anonymous_22(<cmd>:280)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:394)
        at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3090)
        at org.mozilla.javascript.gen._cmd__17.call(<cmd>)
        at org.mozilla.javascript.JavaAdapter.doCall(JavaAdapter.java:606)
        at org.mozilla.javascript.JavaAdapter.access$000(JavaAdapter.java:15)
        at org.mozilla.javascript.JavaAdapter$1.run(JavaAdapter.java:586)
        at org.mozilla.javascript.Context.call(Context.java:489)
        at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:504)
        at org.mozilla.javascript.JavaAdapter.callMethod(JavaAdapter.java:583)
        at adapter1.execute(<adapter>)
        at com.galenframework.TestRunnable.runTest(TestRunnable.java:75)
        at com.galenframework.TestRunnable.run(TestRunnable.java:101)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: org.openqa.selenium.NoSuchWindowException: Unable to find elements on closed window (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'kodagali-PC', ip: '10.0.2.15', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_45'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=false, handlesAlerts=true, version=11, plat
ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:45897/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=true, en
ss}]
Session ID: deba27cf-9dc3-4780-91e4-9a031d96ad4d
*** Element info: {Using=css selector, value=div[data-tab="buy"]}
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
        at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
        at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:595)
        at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:373)
        at org.openqa.selenium.remote.RemoteWebDriver.findElementsByCssSelector(RemoteWebDriver.java:441)
        at org.openqa.selenium.By$ByCssSelector.findElements(By.java:437)
        at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:336)
        at com.galenframework.page.selenium.ByChain.findElement(ByChain.java:63)
        at com.galenframework.page.selenium.SeleniumPage.driverFindElement(SeleniumPage.java:115)
        at com.galenframework.page.selenium.SeleniumPage.locatorToElement(SeleniumPage.java:129)
        at com.galenframework.page.selenium.SeleniumPage.getObject(SeleniumPage.java:77)
        at com.galenframework.page.selenium.SeleniumPage.getObject(SeleniumPage.java:86)
        at com.galenframework.validation.PageValidation.findPageElement(PageValidation.java:89)
        at com.galenframework.validation.specs.SpecValidationSize.check(SpecValidationSize.java:31)
        at com.galenframework.validation.specs.SpecValidationSize.check(SpecValidationSize.java:27)
        at com.galenframework.validation.PageValidation.check(PageValidation.java:71)
        at com.galenframework.validation.PageValidation.check(PageValidation.java:61)
        at com.galenframework.validation.SectionValidation.checkObject(SectionValidation.java:165)
        at com.galenframework.validation.SectionValidation.checkObjects(SectionValidation.java:75)
        at com.galenframework.validation.SectionValidation.checkSection(SectionValidation.java:109)
        at com.galenframework.validation.SectionValidation.checkPageSection(SectionValidation.java:54)
        at com.galenframework.validation.SectionValidation.check(SectionValidation.java:47)
        at com.galenframework.api.Galen.checkLayoutForPage(Galen.java:117)
        at com.galenframework.api.Galen.checkLayout(Galen.java:90)
        at com.galenframework.api.Galen.checkLayout(Galen.java:73)
        at com.galenframework.javascript.GalenJsApi.checkLayout(GalenJsApi.java:100)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:126)
        ... 31 more

========================================
----------------------------------------
========================================
Failed tests:
    Chimera Layout Tests

Suite status: FAIL
Total tests: 1
Total failed tests: 1
Total failures: 1

**Comment by hypery2k** [comment]
was fixed in Galen 2.0.0, You can not use the same specs for Version 1.6. and 2.x as the Spec Syntax has changed, see http://galenframework.com/docs/specslang-2.0-difference/

**Comment by sachinkodagali** [comment]
We had used the updated specs for 2.0 and ran the test on IE11 ..
